v0/destinations/vero/transform.js

```diff
@@ -0,0 +1,124 @@
+const set = require("set-value");
+const get = require("get-value");
+const {
+  defaultRequestConfig,
+  CustomError,
+  constructPayload,
+  getValueFromMessage,
+  isDefinedAndNotNull,
+  removeUndefinedAndNullAndEmptyValues
+} = require("../../util");
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+
+// This function handles the common response payload for the supported calls
+const commonResponseBuilder = (message, category, destination) => {
+  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  const authToken = get(destination, "Config.authToken");
+  set(payload, "auth_token", authToken);
+
+  // channels is defined only for Identify call
```

Comment:
Non-blocking but for simple json key-value setting dot notion or [] can be used for better performance

Reply:
Done